watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1690/amo_1690.py

```diff
@@ -0,0 +1,582 @@
+    m.fs.feed.conc_mass_comp[0, "ammonium_as_nitrogen"].fix(conc_mass_nh4)
+
+    solve(m.fs.feed)
+
+    m.fs.cmf.load_parameters_from_database(use_default_removal=True)
+    m.fs.ad.load_parameters_from_database(use_default_removal=True)
+    m.fs.me.load_parameters_from_database(use_default_removal=True)
+
+
+def initialize_system(m):
+    seq = SequentialDecomposition()
+    seq.options.tear_set = []
+    seq.options.iterLim = 1
+    seq.run(m, lambda u: u.initialize())
+
+
+def solve(blk, solver=None, tee=False, check_termination=True):
+    if solver is None:
+        solver = get_solver()
+    results = solver.solve(blk, tee=tee)
+    if check_termination:
+        assert_optimal_termination(results)
+    return results
+
+
+def display_results(m):
+    print("\n+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+    print("++++++++++++++++++++ DISPLAY RESULTS ++++++++++++++++++++")
+    print("+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+
+    unit_list = ["cmf", "ad", "me"]
+    for u in unit_list:
+        m.fs.component(u).report()
+
+    print("\n---------- Feed volumetric flowrate ----------")
+    feed_vol_flow = value(
+        pyunits.convert(m.fs.feed.flow_vol[0], to_units=pyunits.m**3 / pyunits.hr)
+    )
+    print(f"{feed_vol_flow:.0f} m^3/hr")
+
+    print("\n---------- Feed water concentrations ----------")
+    tss_feed_conc = value(
+        pyunits.convert(
+            m.fs.feed.conc_mass_comp[0, "tss"], to_units=pyunits.mg / pyunits.L
+        )
+    )
+    print(f"TSS: {tss_feed_conc:.1f} mg/L")
+
+    cod_feed_conc = value(
+        pyunits.convert(
+            m.fs.feed.conc_mass_comp[0, "cod"], to_units=pyunits.mg / pyunits.L
+        )
+    )
+    print(f"COD: {cod_feed_conc:.1f} mg/L")
+
+    tkn_feed_conc = value(
+        pyunits.convert(
+            m.fs.feed.conc_mass_comp[0, "tkn"], to_units=pyunits.mg / pyunits.L
+        )
+    )
+    print(f"TKN: {tkn_feed_conc:.1f} mg/L")
+
+    aa_feed_conc = value(
+        pyunits.convert(
+            m.fs.feed.conc_mass_comp[0, "acetic_acid"], to_units=pyunits.mg / pyunits.L
+        )
+    )
+    print(f"Acetic acid: {aa_feed_conc:.1f} mg/L")
+
+    nh4_feed_conc = value(
+        pyunits.convert(
+            m.fs.feed.conc_mass_comp[0, "ammonium_as_nitrogen"],
+            to_units=pyunits.mg / pyunits.L,
+        )
+    )
+    print(f"TSS: {nh4_feed_conc:.1f} mg/L")
+
+    print("\n---------- Anaerobic digester results ----------")
+    biogas_production = value(
+        pyunits.convert(
+            m.fs.ad.biogas_production[0], to_units=pyunits.m**3 / pyunits.hr
+        )
+    )
+    print(f"Biogas production: {biogas_production:.2f} m^3/hr")
+
+    byproduct_tss_rate = value(
+        pyunits.convert(
+            m.fs.ad_byproduct.flow_mass_comp[0, "tss"], to_units=pyunits.kg / pyunits.hr
+        )
+    )
+    print(f"TSS byproduct production: {byproduct_tss_rate:.2f} m^3/hr")
```

Comment:
```suggestion
    print(f"TSS byproduct production: {byproduct_tss_rate:.2f} kg/hr")
```

Reply:
Good eye finding these. Addressed in latest commit.